### PR TITLE
Add default testing group to navigator

### PR DESCRIPTION
**Pull request by nikodamn**
Explicit change of URL in Mercury navigation due to experiment described in DAT-4483

Reviewers:
@Wikia/west-wing @tomasznapieralski

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 1]
```diff
@@ -7,12 +7,21 @@
...
+   */
+
+  private static final String defaultTestingGroup =
+      "&ABTEST.MERCURY_VIEWABILITY_EXPERIMENT=DEFAULT";
+
   public Navigate toPage(String pageName) {
     String host = UrlBuilder.getHostForWiki();
     String cacheBuster = pageName.equals("") ||
                          pageName.equals("/") ? "" : "?cb=" + DateTime.now().getMillis();
+    String abGroup = pageName.equals("") || pageName.equals("/") ? "" : defaultTestingGroup;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: MAJOR Move the "/" string literal on the left side of this string comparison. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1132)
MAJOR Move the "" string literal on the left side of this string comparison. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1132)

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 2]
```diff
@@ -7,12 +7,21 @@
 
 public class Navigate extends BasePageObject {
 
+  /**
+   * Adding default testing group is invoked explicitly in URL as temporary change until
+   * the end of experiment (01.07.2016) described in DAT-4483.
+   */
+
+  private static final String defaultTestingGroup =
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: MAJOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 3 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java (hunk 1 above)
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: if you will navigate to page with `/` you will get `http://www.example.com/&ABTEST.`
add your parameter to the end of line above

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 3]
```diff
@@ -7,12 +7,21 @@
...
+      "&ABTEST.MERCURY_VIEWABILITY_EXPERIMENT=DEFAULT";
+
   public Navigate toPage(String pageName) {
     String host = UrlBuilder.getHostForWiki();
     String cacheBuster = pageName.equals("") ||
                          pageName.equals("/") ? "" : "?cb=" + DateTime.now().getMillis();
+    String abGroup = pageName.equals("") || pageName.equals("/") ? "" : defaultTestingGroup;
 
-    driver.get("http://" + host + pageName + cacheBuster);
+    driver.get("http://" + host + pageName + cacheBuster + abGroup);
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: if you apply comment above, you won't need that

**Comment by tomasznapieralski** [comment]
please change all methods there (you've missed one)

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java (hunk 1 above)
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: good catch, thanks!

**Comment by tomasznapieralski** [comment]
LGTM, you are free to merge :)

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 4]
```diff
@@ -7,10 +7,19 @@
 
 public class Navigate extends BasePageObject {
 
+  /**
+   * Adding default testing group is invoked explicitly in URL as temporary change until
+   * the end of experiment (01.07.2016) described in DAT-4483.
+   */
+
+  private static final String defaultTestingGroup =
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: MAJOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request merged**